message_persistence/message_persistence.md

```diff
@@ -1,10 +1,13 @@
 Message Persistence
 ===================
 
-- ![](/badge/rec.png) A client should provide an option to save its message
+- ![](/badge/req.png) A client should provide an option to save its message
```

Comment:
Well then, if UX isn't the main focus, and this standard distances self from it, I guess I can revert this change - just want to confirm - you want that, right?